frontend/app/mocks/db.ts

```diff
@@ -280,79 +261,32 @@ db.maritalStatus.create({
 });
 
 // seed country list
-db.country.create({
-  countryId: 'CAN',
-  nameEnglish: 'Canada',
-  nameFrench: 'Canada',
-});
-
-db.country.create({
-  countryId: 'USA',
-  nameEnglish: 'United States of America',
-  nameFrench: "États-Unis d'Amérique",
-});
-
-db.country.create({
-  countryId: 'MEX',
-  nameEnglish: 'Mexico',
-  nameFrench: 'Mexique',
-});
-
-// seed province list
-db.region.create({
-  provinceTerritoryStateId: 'ON',
-  countryId: 'CAN',
-  nameEnglish: 'Ontario',
-  nameFrench: 'Ontario',
-});
-
-db.region.create({
-  provinceTerritoryStateId: 'MB',
-  countryId: 'CAN',
-  nameEnglish: 'Manitoba',
-  nameFrench: 'Manitoba',
-});
-
-db.region.create({
-  provinceTerritoryStateId: 'QC',
-  countryId: 'CAN',
-  nameEnglish: 'Quebec',
-  nameFrench: 'Québec',
-});
-
-db.region.create({
-  provinceTerritoryStateId: 'NS',
-  countryId: 'CAN',
-  nameEnglish: 'Nova Scotia',
-  nameFrench: 'Nouvelle-Écosse',
-});
-
-db.region.create({
-  provinceTerritoryStateId: 'NL',
-  countryId: 'CAN',
-  nameEnglish: 'Newfoundland and Labrador',
-  nameFrench: 'Terre-Neuve-et-Labrador',
-});
-
-db.region.create({
-  provinceTerritoryStateId: 'PE',
-  countryId: 'CAN',
-  nameEnglish: 'Prince Edward Island',
-  nameFrench: 'Île-du-Prince-Édouard',
-});
-
-db.region.create({
-  provinceTerritoryStateId: 'UT',
-  countryId: 'USA',
-  nameEnglish: 'Utah',
-  nameFrench: 'Utah',
-});
-
-db.region.create({
-  provinceTerritoryStateId: 'NY',
-  countryId: 'USA',
-  nameEnglish: 'New York',
-  nameFrench: 'État de New York',
-});
+countriesJson.value.forEach((country) =>
+  db.country.create({
+    countryId: country.esdc_countryid,
+    countryCode: country.esdc_countrycodealpha3,
+    nameEnglish: country.esdc_nameenglish,
+    nameFrench: country.esdc_namefrench,
+  }),
+);
+
+// seed region list
+regionsJson.value.forEach((region) =>
+  db.region.create({
+    countryId: db.country.findFirst({ where: { countryId: { equals: region._esdc_countryid_value } } })?.countryCode,
+    provinceTerritoryStateId: region.esdc_internationalalphacode,
+    nameEnglish: region.esdc_nameenglish,
+    nameFrench: region.esdc_namefrench,
```

Comment:
The convention in this file is to use `nameEn` and `nameFr`. I think we should stick with that.